dcos/ingress/linkerd-config.yml

```diff
@@ -0,0 +1,31 @@
+# linkerd config for basic http proxy plus ingress
+# automatically routes port 4242 to "webapp"
+
+admin:
+  port: 9990
+
+namers:
+- kind: io.l5d.marathon
+  host: marathon.mesos
+  port: 8080
+  prefix: "/io.l5d.marathon"
+  experimental: true
+
+routers:
+- protocol: http
+  servers:
+  - port: 4140
+    ip: 0.0.0.0
+  baseDtab: >-
+    /marathonId => /#/io.l5d.marathon;
+    /http/1.1/* => /$/io.buoyant.http.domainToPathPfx/marathonId;
+  label: linkerd_proxy
+- protocol: http
+  servers:
+  - port: 4242
+    ip: 0.0.0.0
+  baseDtab: >-
+    /marathonId/* => /#/io.l5d.marathon/webapp;
+    /marathonId   => /#/io.l5d.marathon;
+    /http/1.1/*   => /$/io.buoyant.http.domainToPathPfx/marathonId;
```

Comment:
For the external node, I think it's probably safer to force all requests to the `webapp` app, rather than allowing certain requests that are formulated correctly to bypass.  That way `webapp` is official the front door.

```
/web        => /#/io.l5d.marathon/webapp;
/http/*/*/* => /web;
```